src/wrappers/any.jl

```diff
@@ -179,6 +179,14 @@ end
 
 Base.show(io::IO, status::Status) = print(io, ToString(status))
 
+# build a RayObject from a string
+function ray_object(data::String)
+    data_buffer = LocalMemoryBuffer(Ptr{Nothing}(pointer(data)), sizeof(data), true)
+    meta_buffer = LocalMemoryBuffer(Ptr{Nothing}(), 0, false)
+    nested_refs = CxxWrap.StdVector([ObjectReference()])
```

Comment:
should this be an empty vector instead of a vector with an empty ref?

Reply:
yeah but I think you can pass an empty (typed) vector

```julia
    nested_refs = CxxWrap.StdVector(ObjectReference[])
```

Reply:
I've deleted this now since we don't need to construct a RayObject on the julia side